frontend/src/pages/TriviaGame.js

```diff
@@ -1,15 +1,28 @@
 import Questions from "../components/Questions";
 import styled from "styled-components/macro";
+import { useState } from "react";
 
 export default function TriviaGame({ numberOfQuestions }) {
+  const [visibleSeconds, setVisibleSeconds] = useState(30);
+
   return (
     <>
-      <Header />
-      <Questions numberOfQuestions={numberOfQuestions} />
+      <Header>
+        <p id="seconds"> {visibleSeconds} seconds left</p>
```

Comment:
You could remove the `id` tag here. This looks like a leftover from `getElemebtsById` era :)

Reply:
Yep, correct, forgot to delete! Thanks!